vendorconsent/consent.go

```diff
@@ -63,10 +64,15 @@ type VendorConsents interface {
 
 // ParseString parses a Raw (unpadded) base64 URL encoded string.
 func ParseString(consent string) (VendorConsents, error) {
-	decoded, err := base64.RawURLEncoding.DecodeString(consent)
+	pieces := strings.Split(consent, ".")
+	decoded, err := base64.RawURLEncoding.DecodeString(pieces[0])
```

Comment:
Do we not support or do we not need the disclosed vendors, allowed vendors, or publisher tc sections?

Reply:
I don't think we need them, and we definitely don't need them for pass 1. If needed, we can modify the TCF 2.0 object creator to take the extra pieces.